### PR TITLE
update sendable check

### PR DESCRIPTION
**Pull request by skylarbarrera**
Fixes RNBW-3147

## Dev checklist for QA: what to test
will need to enable actions on read only wallets
All NFTs in skillet.eth should be sendable(show up in send flow), 
hyperloot nft's mylie.eth should be sendable,
poaps should not be sendable( except for a small % on mainnet)

## Final checklist

- [x] Assigned individual reviewers?
- [x] Added labels?
- [x] Added e2e tests? if not please specify why
- [x] If you added new files, did you update the CODEOWNERS file?

### CODE REVIEW

**Review by osdnk — APPROVED** [review]
yea, looks ok, not generic tho, but cannot think of anything better

File: src/parsers/uniqueTokens.js [hunk 1]
```diff
@@ -91,9 +91,8 @@ export const parseAccountUniqueTokens = data => {
             : collection.name,
         id: token_id,
         isSendable:
-          asset_contract.nft_version === '1.0' ||
-          asset_contract.nft_version === '3.0' ||
-          asset_contract.schema_name === 'ERC1155',
+          asset_contract.contract_standard === 'ERC721' ||
+          asset_contract.contract_standard === 'ERC1155',
```
Comment (osdnk) [inline src/parsers/uniqueTokens.js:95]:
```suggestion
          SUPPORTED_SENDABLE_NFT_CONTRACTS.includes(asset_contract.contract_standard)
```

```
const SUPPORTED_SENDABLE_NFT_CONTRACTS = ['ERC721', 'ERC1155'];
```

Comment (skylarbarrera) [reply src/parsers/uniqueTokens.js:95]: got the v1 & v2 endpoints mixed up, since we're gonna be checking 2 diff properties im gonna not do this

**Review by brunobar79 — APPROVED** [review]
Needs rebasing and making sure CI passes, since we have a specific test that sends an ERC-721

**Comment by skylarbarrera** [comment]
merging without QA since my updated changes don't touch effect the same way my initial one's did

**Pull request merged**